src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Shooter extends SubsystemBase {
+  private final CANSparkMax upperShooterMotor;
+  private final CANSparkMAX lowerShooterMotor;
+  /** Creates a new Shooter. */
+  public Shooter(int upperShooterMotorID, int lowerShooterMotorID) {
+    this.upperShooterMotor = new CANSparkMax(upperShooterMotorID, MotorType.kBrushless);
+    this.lowerShooterMotor = new CANSparkMax(lowerShooterMotorID, MotorType.kBrushless);
+    this.upperShooterMotor.setInverted(true);
```

Comment:
actually, this is not necessary for this subsystem because we will want to move notes into the tunnel with the lower wheels without having them be spun out the upper wheels. having them separate is good for this subsystem!